CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ cmake_minimum_required(VERSION 3.18)
 
 # todo - set version to 0.0.0 once confident in automated versioning
 project(Sunshine VERSION 0.22.0
-        DESCRIPTION "Sunshine is a self-hosted game stream host for Moonlight."
+        DESCRIPTION "Self-hosted game stream host for Moonlight"
```

Comment:
This was modified to appease the homebrew audit step. The same complaints were presented by the Macports team previously, and I think the Flatpak team.